lib/build/constants.js

```diff
@@ -0,0 +1,8 @@
+export const SUPPORTED_FRONTEND_FOR_CUSTOM_UI = ["react-custom"];
```

Comment:
i dont quite like that we have a frontend of type react-custom and not just react. Why should the frontend framework name depend on custom or pre built ui?

Reply:
I took the inspiration from the `next` implementation where for different usecases, a different name is used. Also, react and react-custom have different file structure and boilerplate also, hence for future developers, it would be clear that things are different between the react and the `react-custom`  repo. Another thing was if I added a branching inside the react object for the getFrontendOptions function, it was resulting in lot of code changes to accommodate the new structure.